### DISCUSSION TITLE
I am locked out of my admin accounts.

### DISCUSSION THREAD

**Discussion by NotAHackerManz** [Pryaxis/TShock#2664] [General]
So i started a TShock server last Saturday and I can't access the admin account and it told me to go here.

**Comment by hakusaro** [comment]
Hey! Can you try the instructions [on what to do with a missing setup-code.txt file](https://github.com/Pryaxis/TShock/discussions/2325#discussioncomment-772279) and see if that helps? If it doesn't let me know what you're stuck on and we can get you more personalized help!
